packages/react-dom/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -202,53 +225,1074 @@ export function pushTextInstance(
   target.push(stringToChunk(encodeHTMLTextNode(text)), textSeparator);
 }
 
-const startTag1 = stringToPrecomputedChunk('<');
-const startTag2 = stringToPrecomputedChunk('>');
+const styleNameCache: Map<string, PrecomputedChunk> = new Map();
+function processStyleName(styleName: string): PrecomputedChunk {
+  const chunk = styleNameCache.get(styleName);
+  if (chunk !== undefined) {
+    return chunk;
+  }
+  const result = stringToPrecomputedChunk(
+    escapeTextForBrowser(hyphenateStyleName(styleName)),
```

Comment:
Where did this escape happen previously? Can't find it.

Reply:
This is what this comment is about: https://github.com/facebook/react/blob/master/packages/react-dom/src/shared/dangerousStyleValue.js#L20

We used to not escape the whole attribute, then we did again here: https://github.com/facebook/react/blob/master/packages/react-dom/src/server/ReactPartialRenderer.js#L373-L381

And now we don't again.

Reply:
Ah I missed that `quoteAttributeValueForBrowser` also calls escape inside. Sneaky. :D